### PR TITLE
Correct flap / slat speed and remove warning about <product> and fix reversers

### PR DESCRIPTION
**Pull request by legoboyvdlp**

### CODE REVIEW

**Comment by legoboyvdlp** [comment]
More comming, pls don't merge yet

**Comment by legoboyvdlp** [comment]
Ready @YV3399  :)

**Comment by legoboyvdlp** [comment]
Oh, I have perm to merge... ok to do so @YV3399 ?

File: XMLs/ERJ-170-Model.xml [hunk 1]
```diff
@@ -1074,7 +1074,7 @@
 
     <animation>
         <type>translate</type>
-        <object-name>reverser</object-name>
```
Comment (tdammers) [inline XMLs/ERJ-170-Model.xml:1077]: maybe a good idea to rename the `OBJECT`s in the AC to `reverserL` and `reverserR` or sth like that, to make it clearer?

**Comment by YV3399** [comment]
Thanks Lego, great job

**Pull request merged**